aiidalab_widgets_base/structures.py

```diff
@@ -662,7 +662,12 @@ def _make_ase(self, species, positions, smiles):
         """Create ase Atoms object."""
         # Get the principal axes and realign the molecule along z-axis.
         positions = PCA(n_components=3).fit_transform(positions)
-        atoms = Atoms(species, positions=positions, pbc=True)
+        try:
+            atoms = Atoms(species, positions=positions, pbc=False)
+        except KeyError:
```

Comment:
the only thing I kow is
"We removed 720 768 duplicates by comparing reaction strings without atom mapping and an additional 780 reactions, because the SMILES string could not be canonicalized with RDKit,[34](https://pubs.rsc.org/en/content/articlelanding/2018/sc/c8sc02339e#cit34) as the explicit number of valence electrons for one of the atoms was greater than permitted." DOI	https://doi.org/10.1039/C8SC02339E